implementations/app-swift/src/templates/mod.rs

```diff
@@ -0,0 +1,12 @@
+mod Types_swift;
+
+pub struct Template {
+    pub name: &'static str,
+    pub source: &'static str,
+}
+
+pub fn load_templates() -> Vec<Template> {
+    vec!(
+        Types_swift::load(),
```

Comment:
is there any reason for having only the `Types` template? Would it be possible to also have the `Module` template so we can have things separately? Like in plugins

Reply:
I can separate these bindings into Module.swift and Types.swift.

I was just copying how the others are done. To me, it would be ideal to have a separate file for each module.